src/StudyApi/Controllers/V1/Local/LocalController.cs

```diff
@@ -0,0 +1,146 @@
+    private readonly ILogger<LocalController> _logger;
+
+    public LocalController(IParticipantService participantService, IParticipantRepository participantRepository,
+        IUserService userService, ILogger<LocalController> logger)
+    {
+        _participantService = participantService;
+        _participantRepository = participantRepository;
+        _userService = userService;
+        _logger = logger;
+    }
+
+    /// <summary>
+    /// [AllowAnonymous] Login
+    /// </summary>
+    /// <response code="200">When IsSuccess true</response>
+    /// <response code="500">Server side error</response>
+    [AllowAnonymous]
+    [SwaggerResponse(StatusCodes.Status200OK, Type = typeof(Response<UserLoginResponse>))]
+    [SwaggerResponse(StatusCodes.Status500InternalServerError, Type = null)]
+    [HttpPost("UpdateLinkedRecords")]
+    public async Task<IActionResult> UpdateLinkedRecords()
+    {
+        var userRecordErrors = new List<UserRecordError>();
+        var userRecords = _participantRepository.GetAllAsync();
+        await foreach (var userRecord in userRecords)
+        {
+            if (userRecord.Pk.Contains("DELETED#"))
+            {
+                continue;
+            }
+
+            // get a record that has a NhsNumber and a Participant ID, I will therefore know this record in the db has another record that is linked
+            if (userRecord.NhsNumber != null && userRecord.ParticipantId != null)
+            {
+                try
+                {
+                    // check and see if the cognito record for this user has been disabled to confirm this is a linked account
+                    var cognitoRecord = await _userService.AdminGetUserAsync(userRecord.ParticipantId);
+                    if (!cognitoRecord.Enabled)
+                    {
+                        // get the linked account row by passing the ParticipantId into the pk as the new row saves the PK as ParticipantId to link 
+                        var linkedRecord =
+                            await _participantRepository.GetParticipantAsync(
+                                $"PARTICIPANT#{userRecord.ParticipantId}");
+
+                        // delete the linked row
+                        await _participantRepository.DeleteParticipantAsync(linkedRecord);
+
+                        // create a new row with all the same info with LINKED# replacing PARTICIPANT#
+                        linkedRecord.Pk = $"LINKED#{linkedRecord.ParticipantId}";
+                        linkedRecord.Sk = "LINKED#";
+                        await _participantRepository.CreateParticipantAsync(linkedRecord);
+                    }
+                }
+                catch (Exception e)
+                {
+                    // store errors
+                    userRecordErrors.Add(new UserRecordError
+                    {
+                        UserRecord = userRecord,
+                        Error = e.Message
+                    });
```

Comment:
If this is for development use only, might as well capture the whole exception. Add a property `Exception = e`.